### PR TITLE
Fixes bug for queue and edit commands

### PR DESCRIPTION
**Pull request by wongsm7**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/patients/EditPatientDetailsCommand.java [hunk 1]
```diff
@@ -42,6 +42,8 @@
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Patient: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This patient has already been registered.";
+    public static final String MESSAGE_PATIENT_IN_QUEUE = "You cannot edit while patient is in queue.";
+    public static final String MESSAGE_PATIENT_BEING_SERVED = "You cannot edit while patient is being served.";
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/logic/commands/patients/EditPatientDetailsCommand.java]: Please remove the "You" in the error messages

File: src/main/java/seedu/address/logic/commands/staff/EditStaffDetailsCommand.java [hunk 2]
```diff
@@ -42,6 +42,7 @@
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Staff details: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This staff member has already been registered.";
+    public static final String MESSAGE_DOCTOR_ON_DUTY = "You cannot edit while the doctor is on duty";
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/logic/commands/staff/EditStaffDetailsCommand.java]: Please remove the "You" in the error messages

File: src/main/java/seedu/address/logic/commands/patients/EditPatientDetailsCommand.java (hunk 1 above)
Comment (wongsm7) [reply src/main/java/seedu/address/logic/commands/patients/EditPatientDetailsCommand.java]: Fixed

File: src/main/java/seedu/address/logic/commands/staff/EditStaffDetailsCommand.java (hunk 2 above)
Comment (wongsm7) [reply src/main/java/seedu/address/logic/commands/staff/EditStaffDetailsCommand.java]: Fixed

**Pull request merged**